src/views/Header.js

```diff
@@ -34,9 +34,9 @@ type SubViewProps = NavigationSceneRendererProps & {
   onNavigateBack?: () => void,
 };
 
-type Navigation = NavigationScreenProp<NavigationState, NavigationAction>;
+type Navigation = NavigationScreenProp<*, NavigationAction>;
```

Comment:
Probably. It was more of a `hack`, I decided to use `*` because:
a) I don't access its values, so it doesn't really matter to me
b) It was causing an error in DrawerItem that it's incompatible with `<T: *>`. Can be observed in past CircleCI logs. Not sure how to solve it w/o touching other files.